packages/platform/platform-fastify/readme.md

```
@@ -0,0 +1,67 @@
+  <a href="https://tsed.io/">Website</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://tsed.io/getting-started/">Getting started</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://api.tsed.io/rest/slack/tsedio/tsed">Slack</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://twitter.com/TsED_io">Twitter</a>
+</div>
+
+<hr />
+
+A package of Ts.ED framework. See website: https://tsed.io/getting-started/
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)